core/client/src/main/java/alluxio/client/block/stream/UnderFileSystemFileOutStream.java

```diff
@@ -39,7 +39,7 @@
   public UnderFileSystemFileOutStream(FileSystemContext context, InetSocketAddress address,
       long ufsFileId) throws IOException {
     super(PacketOutStream
-        .createNettyPacketOutStream(context, address, -1, ufsFileId, Long.MAX_VALUE,
+        .createNettyPacketOutStream(context, address, -1, ufsFileId, Long.MAX_VALUE, -1,
```

Comment:
is `-1` used because there is no tier associated with the under storage? Its unfortunate there is no invalid value for the tier, ideally we should just leave the tier field unset in this case (similarly for sessionid).